packages/sources/svelte/src/components/VtmnTextInput.svelte

```diff
@@ -42,21 +48,47 @@
    * @type {boolean}
    */
   export let error = false;
+
+  /**
+   * Icon to be displayed
+   * @type {string}
+   */
+  export let icon;
 </script>
 
 {#if labelText}
-  <label class="vtmn-text-input_label" for="{identifier}">{labelText}</label>
+  <label class="vtmn-text-input_label" for={identifier}>{labelText}</label>
 {/if}
-<input
-  class="vtmn-text-input"
-  class:vtmn-text-input--valid={valid}
-  class:vtmn-text-input--error={error}
-  type="text"
-  {identifier}
-  {disabled}
-  {placeholder}
-  {...$$props}
-/>
-{#if helperText}
-  <p class="vtmn-text-input_helper-text">{helperText}</p>
+{#if multiline}
+  <textarea
+    class="vtmn-text-input"
+    class:vtmn-text-input--error={error}
+    class:vtmn-text-input--valid={valid}
+    {identifier}
+    {disabled}
+    {placeholder}
+    {...$$props}
+  />
+{:else}
+  <div class="vtmn-text-input_container">
+    <input
+      class="vtmn-text-input"
+      class:vtmn-text-input--valid={valid}
+      class:vtmn-text-input--error={error}
+      type="text"
+      {identifier}
+      {disabled}
+      {placeholder}
+      {...$$props}
+    />
+    <span class={icon && `vtmx-${icon}`} />
+  </div>
+  {#if helperText}
+    <p
+      class="vtmn-text-input_helper-text"
+      class:vtmn-text-input_helper-text--error={error}
+    >
+      {helperText}
+    </p>
```

Comment:
Nope, it should be outside the if statement of "multiline" like you did with React implementation.